webserv.json

```diff
@@ -511,75 +511,6 @@
                     "value": "200"
                 }
             ]
```

Comment:
This looks like you deleted a test instead of changing or adding tests... Did you export the collectio nfrom thunderclient and save it in the repository?

Reply:
i'm not entirely sure. we had two different webserv test suites. i might have... i can pull the old ones and restore them for next push.